src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION = 0;
+    private static final int AMOUNT = 1;
+    private static final String DATA_SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
It is necessary to divide the program logic into 3 methods:
1. Reading data from a file
2. Create a report from this data
3. Writing a report to a file

And you method getStatistic() should only call these three methods